docs/ruleset_functions/get_building_segment_SWH_bat.md

```diff
@@ -23,7 +23,10 @@ Logic:
 - get the service_water_heating_building_area_type value: `building_segment_swh_bat = building_segment.get("service_water_heating_building_area_type", None)`
 - if building_segment_swh_bat doesn't have service_water_heating_building_area_type, we need to determine the building segment SWH area type by looking at individual SWH uses: `if building_segment_swh_bat is None`
     - create a dictionary that will hold the different types of swh_use_bat_types and the total service water used for the year: `swh_use_dict = {}`
-    - look at each service water heating use id: `for swh_use in find_all(f'$.buildings[*].building_segments[*][?(@.id {building_segment.id}")].zones[*].spaces[*].service_water_heating_uses[*]', rmd):`
+	- create a list of all of the swh uses stored in spaces - note to DEV team - just take the actual objects, not the object references at this stage: `swh_uses_from_spaces = find_all(f'$.buildings[*].building_segments[*][?(@.id == "{building_segment.id}")].zones[*].spaces[*].service_water_heating_uses[*]', rmd)`
+	- create a list of all the swh uses stored at the building segment level: `swh_uses_direct = find_all(f'$.buildings[*].building_segments[*][?(@.id == "{building_segment.id}")].service_water_heating_uses[*]', rmd)`
+
+    - look at each service water heating use in the building: `for swh_use in swh_use_dict + swh_uses_direct:`
```

Comment:
Not seeing where anything is added to swh_use_dict. I see you create an empty dictionary but do not see where it is populated.

Reply:
it should've been `swh_uses_from_spaces`